online-resources/learn-to-code-resources.md

```diff
@@ -4,9 +4,12 @@
 
 * [Code First Girls: Teach Yourself](https://www.codefirstgirls.org.uk/teach-yourself.html)
 * [Free Code Camp](https://learn.freecodecamp.org/)
+* [The Odin Project](https://www.theodinproject.com/home)
```

Comment:
Not heard of this one thanks!

Reply:
It seems like a really good and free full on curriculum site. A bit similar to freeCodeCamp.

Reply:
It seems like a really good and free full on curriculum site. A bit similar to freeCodeCamp.